modules/tinymce/changelog.txt

```diff
@@ -10,6 +10,7 @@ Version 5.6.0 (TBD)
     Fixed an issue where some attributes in table cells were not copied over to new rows or columns #TINY-6485
     Fixed incorrectly removing formatting on adjacent spaces when removing formatting on a ranged selection #TINY-6268
     Fixed some incorrect types in the new TypeScript declaration file #TINY-6413
+    Fixed an issue where menu items would not expand as expected when changing the viewport size #TINY-3321
```

Comment:
Unsure, but maybe `menu items` -> `menus`? Someone else might want to weigh in...

Reply:
Also not sure if we want to mention shadow DOM in the message? Hm.

Reply:
This isn't related with the viewport size either, so that's incorrect. It's related to the menu items collapsing when rendered in a small container.

- [ ] Fix changelog wording to match what this actually is fixing.